### PR TITLE
br: add retry for check multi ingest

### PR DESCRIPTION
**Pull request by glorv**
### What problem does this PR solve?
Do not run multi ingest check if the store is not online. 

### What is changed and how it works?
When do check tikv multi ingest support:
- skip check offline stores
- Add retry for each store if meets error
- Fall back to not support instead of return error if all retry failed and the cluster doesn't contains tiflash.

### Check List 

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

Documentation

### Release note 

```release-note
Do not check multi-ingest when store is offline.
```

### CODE REVIEW

File: br/pkg/lightning/backend/local/local.go [hunk 1]
```diff
@@ -967,25 +967,62 @@ func NewLocalBackend(
...
 					log.L().Info("multi ingest not support", zap.Any("unsupported store", s))
 					local.supportMultiIngest = false
 					return nil
 				}
 			}
-			return errors.Trace(err)
+			// if the cluster contains no TiFlash store, we don't need the multi-ingest feature,
+			// so in this condition, downgrade the logic instead of return an error.
+			if hasTiFlash {
+				return errors.Trace(err)
```
Comment (gozssky) [inline br/pkg/lightning/backend/local/local.go:1017]: Why skip retry when `hasTiFlash` is true?

File: br/pkg/restore/import.go [hunk 2]
```diff
@@ -6,6 +6,7 @@ import (
 	"bytes"
 	"context"
 	"crypto/tls"
+	"github.com/pingcap/kvproto/pkg/metapb"
```
Comment (gozssky) [inline br/pkg/restore/import.go:9]: need fmt

**Review by gozssky — COMMENTED** [review]
rest LGTM

File: br/pkg/lightning/backend/local/local.go (hunk 1 above)
Comment (glorv) [reply br/pkg/lightning/backend/local/local.go:1021]: fixed

**Review by gozssky — APPROVED** [review]

**Review by kennytm — APPROVED** [review]

**Comment by zhangjinpeng1987** [comment]
This PR doesn't have test, please add it. @glorv

**Pull request merged**